Sources/Operators.swift

```diff
@@ -16,6 +16,10 @@ import Foundation
 
 // MARK: - <| Operator
 
+precedencegroup Decodative {
+    associativity: left
```

Comment:
I'll remove this precedence group and we can just stick with the defaults. I'll add a comment line just so we remember what it is doing. Hidden defaults always trip me up.